lib/ansible/modules/cloud/amazon/ssm_send_command.py

```diff
@@ -0,0 +1,211 @@
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: ssm_send_command
+short_description: Execute commands through Simple System Manager (SSM) a.k.a. Run Command
+description:
+  - This module allows you to execute commands through SSM/Run Command.
+version_added: "2.3"
+extends_documentation_fragment:
+  - aws
+author: "Joe Wozniak <woznij@amazon.com>"
+requirements:
+  - python >= 2.6
+  - boto3
+notes:
+  - Async invocation will always return an empty C(output) key.
+  - Synchronous invocation may result in a function timeout, resulting in an
+    empty C(output) key.
+options:
+  name:
+    description:
+      - This should match the name of the SSM document to be invoked.
+    required: true
+    default: NONE
+  comment:
+    description:
+      - A comment about this particular invocation.
+    required: false
+    default: NONE
+  instanceIds:
+    description:
+      - This should contain an array of instance IDs for the instances you
+        wish to run this command document against.
+    required: true
+    default: NONE
+  wait:
+    description:
+      - Whether to wait for the function results or not. If I(wait) is false,
+        the task will not return any results. To wait for the command to
+        complete, set C(wait=true) and the result will be available in the
+        I(output) key.
+    required: false
+    default: true
+  parameters:
+    description:
+      - A dictionary to be provided as the parameters for the SSM document
+        you're invoking.
+    required: false
+    default: {}
+'''
+
+EXAMPLES = '''
+- ssm_send_command:
+    name: AWS-RunPowerShellScript
+    comment: "Run inventory script"
+    instanceIds:
+      - i-123987193812
+      - i-289189288278
+    parameters:
+      commands:
+        - "c:\scripts\get-inventory.ps1"
+        - "c:\scripts\cleanup.ps1"
+      workingDirectory:
+        - "c:\scripts"
+      executionTimeout:
+        - "600"
+    wait: true
+  register: response
+'''
+
+RETURN = '''
+output:
+    description: If wait=true, will return the status of sent command.
+    returned: success
+    type: dict
+    sample: "{ 'output': 'something' }"
+status:
+    description: C(Status) of AWS SSM API call
+    type: str
+    sample: Success
+'''
+
+from ansible.module_utils.basic import AnsibleModule, BOOLEANS
+from ansible.module_utils.ec2 import boto3_conn, ec2_argument_spec, get_aws_connection_info
+import traceback
+from time import sleep
+
+try:
+    import boto3
+    import botocore
+    HAS_BOTO3 = True
+except ImportError:
+    HAS_BOTO3 = False
+
+
+def main():
+    argument_spec = ec2_argument_spec()
+    argument_spec.update(dict(
+        name                 = dict(required=True),
```

Comment:
You should try to follow pep8.  Therefore, the spacing here isn't required.  Also, applies to the rest of the dict